clang/include/clang/Tooling/Tooling.h

```diff
@@ -223,7 +223,11 @@ buildASTFromCode(StringRef Code, StringRef FileName = "input.cc",
 /// \param PCHContainerOps The PCHContainerOperations for loading and creating
 /// clang modules.
 ///
-/// \param Adjuster A function to filter the command line arguments as specified.
+/// \param Adjuster A function to filter the command line arguments as
+/// specified.
+///
+/// \param FileSystem FileSystem for managing and looking up files.
```

Comment:
can we refer to this as `BaseFS` instead and mention that `VirtualMappedFiles` will take precedence over contents in `BaseFS`?

Reply:
Done.